src/kirby/components/page/page.component.html

```diff
@@ -5,51 +5,55 @@
         text=""
         [defaultHref]="defaultBackHref"
         icon="assets/kirby/icons/svg/arrow-back.svg"
-        *ngIf="!hideBackButton"
+        [style.visibility]="!hideBackButton ? 'visible' : 'hidden'"
```

Comment:
Consider flipping for readability + not setting style attribute if visible:
```suggestion
        [style.visibility]="hideBackButton ? 'hidden' : null"
```